refactoring.api/src/org/netbeans/modules/refactoring/api/RefactoringSession.java

```diff
@@ -210,34 +189,32 @@ public void start(ProgressEvent event) {
 
         @Override
         public void step(ProgressEvent event) {
-            current = current + progressStep;
+            current += progressStep;
             fireProgressListenerStep((int) current);
         }
 
         @Override
         public void stop(ProgressEvent event) {
-            // do not rely on plugins;
+            /* do not rely on plugins; */
```

Comment:
Why the change to a multiline comment?

Reply:
If using the toggle comment feature with a hotkey, it will use single line comments on each line of the whole selected block. So, the thought was, that a multi-line comment would better reflect the true nature of a comment, and not just some blending out.

Reply:
Line comment was specifically invented for realy short notes and it's faster to type than /* - */ pair. I don't see a point to change it for one-liners. Maybe for long multilines.
It would be more useful to extend the toggle comment feature to:
- make a block comment (maybe a different hotkey or an additional modifier)
- turn block of line comments into block comment or vice versa

Whether to use multilines or block comments depend on personal or group development style - it's not a defect to fix.

Reply:
A different hotkey is not a good idea for me. I have reconfigured some actions to use the F1 to F12 keys. And since that is just not enough, I have started to use the Ctrl key in combination with some characters. Just so, that I can easily reach them both with only my left hand, and without broken fingers. At this side of the keyboard there aren't much unused keys left for reconfiguration.  And it would feel unnatural to use two different keystrokes for similar actions. The /**/ commentation mode is already used for documentation. Why not keep it like that, and use // mostly for partial code changes? Since "do not rely on plugins;" can not be evaluated by the compiler, I would recomment using /**/ commentation mode.